### ISSUE TITLE
iptsd seems to register large touch points mirrored?

### ISSUE THREAD

**Issue by makinbacon21** [linux-surface/iptsd#108]
![image](https://user-images.githubusercontent.com/19865598/218780999-d06ed6e6-693e-4a55-bd25-8b26cf436e73.png)

This may be a bug in `iptsd-show`, but seems odd. Picture is of my placing the side of my hand on the screen (as if to write with pen).

**Comment by StollD** [comment]
Could you upload a binary log that can reproduce this? I ran a quick test on some old palm data I had lying around but for me it works fine.

![00209](https://user-images.githubusercontent.com/7887606/218788170-9efb7ab0-ff80-4611-81c5-7954d58a214a.png)

```bash
$ sudo iptsd-dump $(iptsd-find-hidraw) data.bin

# Do this to check if the logged data shows the issue
$ sudo iptsd-plot data.bin out/
```

**Comment by makinbacon21** [comment]
It does in fact
![00117](https://user-images.githubusercontent.com/19865598/218789701-13f4b715-8307-4313-bddb-0b96850043b3.png)
![00171](https://user-images.githubusercontent.com/19865598/218789724-037ea9c0-f90b-4e9f-bbc2-28681eabd0b2.png)

Binary Log: https://mega.nz/file/L34kBC4I#dWiqcK7Jk6nlgr5ibNqS7VE3m-0ptaG9mtFpVNUNG5g

**Comment by makinbacon21** [comment]
I'm on SLS (with ithc-linux) for reference

**Comment by StollD** [comment]
Because I am an idiot I hardcoded the settings for my SB2 into the code that handles the orientation of a contact instead of using the proper device specific variables.

Should be fixed with https://github.com/linux-surface/iptsd/commit/272d815c018a246784fdf398ec3a9c11c9f85587

**Comment by StollD** [comment]
For reference: 

![00171](https://user-images.githubusercontent.com/7887606/218816347-07554d80-a569-4b4e-8136-ee951034176f.png)

**Comment by StollD** [comment]
The fix has been released with iptsd 1.1.0
